IPython/terminal/ptshell.py

```diff
@@ -49,8 +49,7 @@ def get_completions(self, document, complete_event):
         )
         start_pos = -len(used)
         for m in matches:
-            yield Completion(m, start_position=start_pos)
-
+            yield Completion(m, start_position=start_pos, display_meta=m[::-1])
```

Comment:
completion.py in Python Prompt Toolkit [[link](https://github.com/jonathanslenders/python-prompt-toolkit/blob/master/prompt_toolkit/completion.py)]